### DISCUSSION TITLE
Unit Test

### DISCUSSION THREAD

**Discussion by gitonio** [evcxr/evcxr#160] [Q&A]
Can I call a rust unit test from a jupyter notebook? If so how?

**Comment by davidlattimore** [comment]
Interesting question. Probably not, but I'm having trouble imagining how that would work. Can you describe in more detail what the workflow you're thinking might look like?

**Reply by gitonio** [reply]
In python notebook you can call a unit test from the notebook and see test result. The example I saw had a student notebook where the student is required to make the unit test pass. The notebook already has a cell with the call to the unit test in it.
I imagine calling 'cargo test some_test' from the notebook.

**Reply by davidlattimore** [reply]
Are the test and the code-under-test also defined in the notebook?
